docs/source/commands.rst

```diff
@@ -62,7 +62,7 @@ Configuration
 Gohan CLI Client is configured though environment variables. Configuration can be divided into few sections.
 
 You can use default configuraion for client with:
-:code:`source ./etc/gohan_client.rc`
+:code:`source ./etc/gohan_cli.rc`
```

Comment:
Instead of changing docs we should rename the file, because Gohan CLI is not only client but also server, schema validation etc.